target/linux/ipq40xx/files-4.19/arch/arm/boot/dts/qcom-ipq4019-eap2200.dts

```diff
@@ -0,0 +1,284 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "EnGenius EAP2200";
+	compatible = "engenius,eap2200";
+
+	chosen {
+		/*
+		 * Overwrite u-boot mtdparts cmdline parameters
+		 */
+		 bootargs-append = " ubi.mtd=ubi root=/dev/ubiblock0_1";
+	};
+
+	soc {
+		rng@22000 {
+			status = "okay";
+		};
+
+		mdio@90000 {
+			status = "okay";
+		};
+
+		ess-psgmii@98000 {
+			status = "okay";
+		};
+
+		crypto@8e3a000 {
+			status = "okay";
+		};
+
+		watchdog@b017000 {
+			status = "okay";
+		};
+
+		ess-switch@c000000 {
+			status = "okay";
+			switch_lan_bmp = <0x10>;
+		};
+
+		edma@c080000 {
+			status = "ok";
+		};
+
+		keys {
+				compatible = "gpio-keys";
+
+				wps {
+					label = "wps";
+					gpios = <&tlmm 18 GPIO_ACTIVE_LOW>;
+					linux,code = <KEY_WPS_BUTTON>;
+				};
+		};
+
+		aliases {
```

Comment:
I moved that to the main "/" node. As this does not really belong into the "soc" sub.